contracts/security/src/core/error.rs

```diff
@@ -0,0 +1,44 @@
+use cosmwasm_std::StdError;
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum ContractError {
+    // General Errors
+    #[error("{0}")]
+    Std(#[from] StdError),
+
+    #[error("Unauthorized")]
+    Unauthorized {},
+
+    #[error("Asset does not exist for address {0}")]
+    AssetDoesNotExist(String),
+
+    #[error("Security type {0} is still in use and cannot be removed")]
+    SecurityInUse(String),
+
+    #[error("Security type {0} is not a valid security type")]
+    InvalidSecurityType(String),
+
+    #[error("Security type {0} has an empty string for category or name.")]
```

Comment:
NIT: have a period at end of this but none of the others?  Should keep it consistent.

Reply:
Thanks for finding the typo.